src/test/java/net/snowflake/ingest/streaming/internal/datatypes/IcebergStructuredIT.java

```diff
@@ -236,6 +244,180 @@ public void testFieldName() throws Exception {
         val);
   }
 
+  @Test
+  public void testExtraFields() throws SQLException {
+    String tableName = createIcebergTable("object(k1 int)");
+    SnowflakeStreamingIngestChannel channel =
+        openChannel(tableName, OpenChannelRequest.OnErrorOption.CONTINUE);
+    Map<String, Object> row = new HashMap<>();
+    row.put("k2", 1);
+    row.put("k.3", 1);
+    row.put("k\\4", 1);
+    InsertValidationResponse insertValidationResponse =
+        channel.insertRow(createStreamingIngestRow(row), UUID.randomUUID().toString());
+    assertThat(insertValidationResponse.getInsertErrors().size()).isEqualTo(1);
+    assertThat(insertValidationResponse.getInsertErrors().get(0).getExtraColNames())
+        .containsOnly("VALUE.k2", "VALUE.k\\.3", "VALUE.k\\\\4");
```

Comment:
FOR KC team: The first VALUE is the default column name used by the test harness, and doesn't have any semantics. This is unlike the key-value/list/element/value words that you'll see in the examples that follow.